### ISSUE TITLE
Could you share your final mAP results?

### ISSUE THREAD

**Issue by QiuJueqin** [kimberninger/SqueezeDet2#1]
Hi, I noticed that you modified both positive (3.75) and nagative (5.0) weights of the confidence score loss from the official repository, would you mind sharing your final mAP results with these modification?

I also believe that the loss function in the official repository undergoes some error, where the batch size was not correctly taken into account.

**Comment by kimberninger** [comment]
Hi and thanks for your question and your interest in this repo!

Indeed we achieve the exact same results with our model as with the original since we took extra caution to precisely re-implement it in Keras without changing its basic functionality.
But you are right, we too recognized the error in the original code and adapted the loss normalization in order to account for the batch size.

The original authors use weights 75 and 100 for the respective losses and a batch size of 20, so we simply normalized the hyper-parameters accordingly to 3.75 and 5.

Another difference you might spot is the choice NMS hyper-parameters, as we use the built-in algorithm from the TensorFlow framework which seems to treat the top-n selection of bounding boxes differently. So we reduced it from 64 to 10.

We haven't yet trained the model from ground up but rather used for prediction by loading the provided weights from the original model, resulting in the same predictions.
As soon as we have further insight on the models performance after bootstrapping the training process, we will share the trained model in this repository.
